### PR TITLE
Add StatusCallback options 

### PR DESCRIPTION
**Pull request by acodegirl**
## What problem does the pull request solve?
Added  StatusCallback option to sendEmail & sendSms

## Checklist

- [x] I’ve used the pull request template
- [x] I’ve written unit tests for these changes
- [x] I’ve updated the documentation in
  - `README.md`
- [ ] I’ve bumped the version number in
  - `package.json`
- [ ] I've added new environment variables in
  - `CONTRIBUTING.md`

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -11,16 +11,21 @@ export default class Client {
...
-    options?: any
+    options?: any,
+    statusCallbackUrl?: string,
+    statusCallbackBearerToken?: string
   ): any {
     options = options || {};
-    const err = checkOptionsKeys(
+    statusCallbackUrl = statusCallbackUrl || "";
+    statusCallbackBearerToken = statusCallbackBearerToken || "";
+    let err = checkOptionsKeys(
```
Comment (jonathaningram) [inline src/client.ts:21]: `const err`

File: src/client.ts [hunk 2]
```diff
@@ -11,16 +11,21 @@ export default class Client {
...
     emailAddress: string,
-    options?: any
+    options?: any,
+    statusCallbackUrl?: string,
+    statusCallbackBearerToken?: string
   ): any {
     options = options || {};
-    const err = checkOptionsKeys(
+    statusCallbackUrl = statusCallbackUrl || "";
+    statusCallbackBearerToken = statusCallbackBearerToken || "";
```
Comment (jonathaningram) [inline src/client.ts:20]: should these go under `options`—what was the reason to make them separate parameters to this function?

Comment (acodegirl) [reply src/client.ts:20]: I thought options are related to template & email/ sms mandatory params. 
And didn't think that its to categorize all optional values. So I added seperately.

Comment (jonathaningram) [reply src/client.ts:20]: yeah they should probably just go under `options`. they are just as equal as the other generic options like `personalisation`, `reference`, `smsSenderId`. @dagoof thoughts?

Comment (jonathaningram) [reply src/client.ts:20]: having additional parameters after the generic `options` looks odd to me too

Comment (dagoof) [reply src/client.ts:20]: put it under options. it's not any different to the other parameters

Comment (dagoof) [reply src/client.ts:20]: is there a reason we default to empty string? I would expect us to just not include the parameter if not present

Comment (jonathaningram) [reply src/client.ts]: yeah that seems sensible to me to exclude if empty. and yeah @acodegirl let's go with putting these under `options`

**Pull request merged**